groups/instance/main.tf

```diff
@@ -1,10 +1,10 @@
 terraform {
-  required_version = ">= 0.13.0, < 0.14"
+  required_version = ">= 1.3.0, < 2.0.0"
 
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 3.0, < 4.0"
+      version = ">= 5.0, < 6.0"
     }
   }
```

Comment:
We should additionally declare the `cloudinit` and `vault` providers within the `required_providers` block

E.g.:
```
  required_providers {
    aws = {
      source  = "hashicorp/aws"
      version = ">= 5.0, < 6.0"
    }
    cloudinit = {
      source  = "hashicorp/cloudinit"
      version = ">= 2.3.0, < 3.0.0"
    }
    vault = {
      source  = "hashicorp/vault"
      version = ">= 3.23.0, < 4.0.0"
    }
  }

Reply:
Implemented as required